api/src/test/java/uk/gov/dhsc/htbhf/claimant/service/EligibilityAndEntitlementServiceTest.java

```diff
@@ -154,20 +159,42 @@ void shouldEvaluateClaimForGivenPaymentCycle() {
         given(client.checkIdentityAndEligibility(any())).willReturn(IDENTITY_AND_ELIGIBILITY_RESPONSE);
         given(paymentCycleEntitlementCalculator.calculateEntitlement(any(), any(), any(), any())).willReturn(VOUCHER_ENTITLEMENT);
         given(eligibilityAndEntitlementDecisionFactory.buildDecision(any(), any(), anyBoolean())).willReturn(decision);
-        Claimant claimant = paymentCycle.getClaim().getClaimant();
+        Claim claim = paymentCycle.getClaim();
         LocalDate cycleStartDate = LocalDate.now();
 
         //When
-        EligibilityAndEntitlementDecision result = eligibilityAndEntitlementService.evaluateClaimantForPaymentCycle(claimant, cycleStartDate, paymentCycle);
+        EligibilityAndEntitlementDecision result = eligibilityAndEntitlementService.evaluateClaimantForPaymentCycle(claim, cycleStartDate, paymentCycle);
 
         //Then
         assertThat(result).isEqualTo(decision);
-        verify(client).checkIdentityAndEligibility(claimant);
+        verify(client).checkIdentityAndEligibility(claim.getClaimant());
         verify(paymentCycleEntitlementCalculator)
                 .calculateEntitlement(Optional.of(EXPECTED_DELIVERY_DATE_IN_TWO_MONTHS), MAGGIE_AND_LISA_DOBS, cycleStartDate, VOUCHER_ENTITLEMENT);
         verify(eligibilityAndEntitlementDecisionFactory).buildDecision(IDENTITY_AND_ELIGIBILITY_RESPONSE, VOUCHER_ENTITLEMENT, false);
     }
 
+    @Test
+    void shouldEvaluateClaimForGivenPaymentCycleWithEligibilityOverride() {
+        //Given
+        Claim claim = ClaimTestDataFactory.aValidClaimBuilder().eligibilityOverrideOutcome(EligibilityOutcome.CONFIRMED).build();
```

Comment:
Might be worth a new method on `ClaimTestDataFactory` for this, something like `aValidClaimWithOverride` or similar.

Reply:
TODO statement added